mailbox/plugin/deleted-messages-vault/src/main/java/org/apache/james/vault/DeletedMessageZipper.java

```diff
@@ -43,6 +43,8 @@
         Optional<InputStream> load(DeletedMessage deletedMessage);
     }
 
+    static final String ZIPPED_FILE_EXTENSION = ".eml";
```

Comment:
is `.zip`?

Reply:
No, it is the file extensions for what is stored in the zip.